mitiq/calibration/settings.py

```diff
@@ -81,6 +82,20 @@ def most_likely_bitstring(self) -> str:
     def largest_probability(self) -> float:
         return max(self.ideal_distribution.values())
 
+    def converted_circuit(self, circuit_type: str) -> QPROGRAM:
+        """Adds measurements to all qubits and convert
+        to the input frontend type.
+
+        Args:
+            circuit_type: The circuit type as a string.
+                For supported circuit types see mitiq.SUPPORTED_PROGRAM_TYPES.
+        Returns:
+            The converted circuit with final measurements.
+        """
+        circuit = self.circuit.copy()
+        circuit.append(cirq.measure(circuit.all_qubits()))
+        return convert_from_mitiq(circuit, circuit_type)
```

Comment:
I like this small, modular function! Nice job breaking stuff up into manageable bits.

Reply:
Actually I wasn't sure if keeping this method or removing it, given that now we have a cirq_executor in the Calibrator.  So this method is now not essential.
I think it doesn't harm and it may help if the user wants to apply a more low-level and manual workflow.